core/tests/protractor_utils/AdminPage.js

```diff
@@ -266,6 +276,60 @@ var AdminPage = function() {
         });
       });
   };
+  var _assignReviewer = function(
+      username, reviewCategory, languageDescription = null) {
+    waitFor.elementToBeClickable(
+      adminRolesTab, 'Admin Roles tab is not clickable');
+    adminRolesTab.click();
+
+    waitFor.visibilityOf(
+      assignReviewerForm, 'Assign reviewer form is not visible');
+
+    var usernameInputField = assignReviewerForm.element(reviewerUsernameCss);
+    waitFor.visibilityOf(
+      usernameInputField,
+      'Username input field is not visible in assign reviewer form');
+    usernameInputField.sendKeys(username);
+
+    var reviewCategorySelectField = assignReviewerForm.element(
```

Comment:
Can we use "role" instead of "category" here to be consistent with what is shown on the admin page?

Reply:
We have used this terminology for review as the role is "reviewer" and a reviewer role can be in different `categories`.